tests/python/unittest/test_symbol.py

```diff
@@ -115,7 +115,7 @@ def test_symbol_infer_shape_var():
     shape = (2, 3)
     a = mx.symbol.Variable('a', shape=shape)
     b = mx.symbol.Variable('b')
-    c = a+b
+    c = mx.symbol.ewise_plus(a, b)
```

Comment:
before merging broadcast_plus to + the above code works and now it doesn't.
This is a breaking change and we should only do it if its absolutely necessary.

If we are going through with this, we need better naming. There should be +, plus, and broadcast_plus and this should be done for every op, not just plus.